api/sign.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+var sjcl = require('ripple-lib').sjcl;
+var Seed = require('ripple-lib').Seed;
+var SerializedObject = require('ripple-lib').SerializedObject;
+var validate = require('./lib/validate');
+
+/**
+ * These prefixes are inserted before the source material used to
+ * generate various hashes. This is done to put each hash in its own
+ * "space." This way, two different types of objects with the
+ * same binary data will produce different hashes.
+ *
+ * Each prefix is a 4-byte value with the last byte set to zero
+ * and the first three bytes formed from the ASCII equivalent of
+ * some arbitrary string. For example "TXN".
+ */
+var HASH_TX_ID = 0x54584E00; // 'TXN'
+var HASH_TX_SIGN = 0x53545800; // 'STX'
+var HASH_TX_SIGN_TESTNET = 0x73747800; // 'stx'
+
+function getPublicKey(address, secret) {
```

Comment:
getPublicKey and getPrivateKey are exactly the same.

Maybe you could get away with just the one, getKeyPair?